pandas/core/frame.py

```diff
@@ -3851,6 +3848,7 @@ def _getitem_multilevel(self, key):
                     top = top[0]
                 if top == "":
                     result = result[""]
+                    # result = result.squeeze("columns") # TODO iloc?
```

Comment:
Can you explain this potential TODO?

Reply:
whoops, my bad, this was a note for me as I was testing stuff - at first trying to avoid the `result = result[""]` to avoid the code path with `self.values`. Will remove that.